source/assets/stylesheets/components/_header.scss

```diff
@@ -1,4 +1,4 @@
-.header {
+header, .header {
```

Comment:
Same remark as here https://github.com/brug-be/rubybelgium/pull/82/files#diff-41aca4d97598d89461ad2e4d26d4fd39R33